07-react.js/react-boilerplate-master/components/title-component.js

```diff
@@ -0,0 +1,12 @@
+// VENDOR LIBS
```

Comment:
You have reason, but i created that component only for show something when i started with react. It is a good idea use a props for that.